discord_slash/context.py

```diff
@@ -373,33 +373,57 @@ async def edit_origin(self, **fields):
         """
         _resp = {}
 
-        content = fields.get("content")
-        if content:
-            _resp["content"] = str(content)
+        try:
+            content = fields["content"]
+        except KeyError:
+            pass
+        else:
+            if content is not None:
+                content = str(content)
+            _resp["content"] = content
+
+        try:
+            components = fields["components"]
+        except KeyError:
+            pass
+        else:
+            if components is None:
+                _resp["components"] = []
+            else:
+                _resp["components"] = components
```

Comment:
Would it not make sense to duplicate this code block for `embeds`

This would avoid "Provide a list of embeds" errors when passing None in the embeds field, which would likely mean the user wants to remove the embeds

Reply:
Hmmm... probably? What's interesting, though, is that discord.py _doesn't_ allow for that type of behavior, as you can see [here](https://github.com/Rapptz/discord.py/blob/462ba84809f43349296c44cbe6468f631a00edab/discord/webhook.py#L1046) in the v1.x branch in `webhooks.py`.

I believe 2.0 also doesn't allow for this behavior? You can take a look [here](https://github.com/Rapptz/discord.py/blob/d30fea5b0dcba9cd130026b56ec01e78bd788aff/discord/webhook/async_.py#L440), but passing in a `None` value there would error out the code, I believe.

Reply:
I'm curious what their rationale behind that would have been. But oh well, its probably best if we mirror d.py's behaviour